lib/beaker/logger.rb

```diff
@@ -312,6 +312,13 @@ def Logger.generate_dated_log_folder(base_dir, timestamp)
       log_dir
     end
 
+    #Remove color codes from provided string.  Color codes are of the format /(\e\[\d\d;\d\dm)+/.
+    #@param [String] text The string to remove color codes from
+    #@return [String] The text without color codes
+    def Logger.strip_color_codes(text)
```

Comment:
Shouldn't this just be called 'strip_color_codes' instead of 'Logger.strip_color_codes'?  Looks like a copy/paste error.